src/Code.ts

```diff
@@ -209,12 +210,12 @@ export function deepGenerate(object: unknown): string {
   return result;
 }
 
-const configPromise = resolveConfig('./');
+const configPromise = resolveConfig?.('./');
```

Comment:
Huh, interesting. Sgtm. Can you include an in-source comment here that mentions the CDN/Deno use case and maybe links to this PR?